runtime/nls/port/port.nls

```diff
@@ -339,12 +339,12 @@ J9NLS_PORT_MMAP_INVALID_FILE_HANDLE.user_response=Contact your service represent
 
 # END NON-TRANSLATABLE
 
-J9NLS_PORT_LINUXDUMP_PIPE_CORE=%s setting \"%s\" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.%d.\n
+J9NLS_PORT_LINUXDUMP_PIPE_CORE=%s setting \"%s\" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.%d.  Review the manual for the external program to find where the core dump is written and ensure the program does not truncate it.\n
```

Comment:
By changing an existing message, only the English output will have this change and other languages will not until the next translation cycle next year. I may be better to add a new message that will output in English for all locales until it's translated.

Reply:
If I understand correctly, do you mean add a new message and use that in place of JVMPORT030W? If so, I worry that some users might be watching for JVMPORT030W as a symptom of a core dump and that check would no longer work.

Reply:
There is a reference in `J9NLS_VM_CHECK_DUMP_COREPATTERN_PIPE.system_action` that should be updated.